src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard-area-chart/dashboard-area-chart.component.ts

```diff
@@ -276,15 +314,17 @@ export class DashboardAreaChartComponent implements OnChanges {
     let maxValueDataUnits = '';
     let extraRoom = 1.2;
 
-    if (this.data) {
-      let maxValueData = Math.max(...this.data.map((values: any) => values[1]));
-      if (this.data2) {
-        let maxValueData2 = Math.max(...this.data2.map((values: any) => values[1]));
-        maxValue = Math.max(maxValueData, maxValueData2);
-      } else {
-        maxValue = maxValueData;
+    if (this.dataArray) {
+      const allValues = this.dataArray
+        .filter(Array.isArray) // Filter out non-array elements
```

Comment:
We need to check here that there are only "arrays" inside the this.dataArray array..we sometimes get undefined or a blank string inside the dataArray as well..so to avoid the error here we are only filtering the elements which are of type Array in this.dataArray

Reply:
I will do this checks in component itself before sending the data to be plotted. Making sure the empty array is returned if the data is None or undefined.

Reply:
Updated the PR